src/StakeRegistry.sol

```diff
@@ -253,91 +246,117 @@ contract StakeRegistry is VoteWeigherBase, StakeRegistryStorage {
     /**
      * @notice Finds the updated stake for `operator` for `quorumNumber`, stores it and records the update
      * @dev **DOES NOT UPDATE `totalStake` IN ANY WAY** -- `totalStake` updates must be done elsewhere.
-     * @return `int256` The change in the operator's stake as a signed int256
-     * @return `uint96` The operator's new stake after the update
+     * @return delta The change in the operator's stake as a signed int256
+     * @return hasMinimumStake Whether the operator meets the minimum stake requirement for the quorum
      */
     function _updateOperatorStake(
         address operator,
         bytes32 operatorId,
         uint8 quorumNumber
-    ) internal returns (int256, uint96) {
+    ) internal returns (int256 delta, bool hasMinimumStake) {
         /**
          * Get the operator's current stake for the quorum. If their stake
          * is below the quorum's threshold, set their stake to 0
          */
         uint96 currentStake = weightOfOperatorForQuorum(quorumNumber, operator);
         if (currentStake < minimumStakeForQuorum[quorumNumber]) {
```

Comment:
this is not super clean... i think there's an argument that operators should be deregistered from this quorum in this case, but later...

Reply:
They are. There are 2 cases:
- when an operator tries to register and doesn't have the minimum stake, `registerOperator` reverts
- when an operator deregisters, this returns true, but also that's what we're doing

Or are you suggesting something else?